src/state/slices/opportunitiesSlice/selectors/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './lpSelectors'
+export * from './stakingSelectors'
```

Comment:
Just curious, was the rationale to separate these not to clutter the already very cluttered selectors module or was it to minimize circular dependencies risks ?
Either way, I'm all for it 🤌

Reply:
The intent here was to declutter the selectors module. 
I wasn't thinking about minimizing the risk of circular dependencies at the time, but if that's an added bonus then hell yeah brother.